### ISSUE TITLE
Continuous_InitialState uses wrong enumeration

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#805]
**Reported by sjoelund&#46;se on 7 Sep 2012 09:10 UTC**
In ModelicaTest.Blocks.Continuous_InitialState , you have the following modifiers:
```
pID(initType=Modelica.Blocks.Types.Init.InitialState),
limPID(initType=Modelica.Blocks.Types.Init.InitialState),
```

But they take Modelica.Blocks.Types.InitPID as initType.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/805

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 10 Dec 2012 12:26 UTC**
Fixed in 16f734e74b744b7140def20d39a08cc0f4e121cb by replacing the declaration with the correct enumeration (as suggested)

**Comment by modelica-trac-importer** [comment]
**Comment by sjoelund&#46;se on 11 Jan 2013 10:37 UTC**
Added the same fix for Continuous_InitialOutput in 54614a220973287c74ce80fe825c0eff8a9a0c12
